packages/gatsby-theme-store/src/sdk/newsletter/README.md

```diff
@@ -0,0 +1,11 @@
+# Newsletter
+
+For the newsletter to work, it's necessary to create an entity in Master Data with Acronym `NL`.
+
+The entity must contain the fields:
+
+| Name              | Display Name        | Type        |
+|-------------------|---------------------|-------------|
+| email             | email               | Varchar 750 |
+| isNewsletterOptIn | Receber Newsletter? | Boolean     |
+| nome              | nome                | Varchar 750 |
```

Comment:
Our code reference fields in Portuguese?

Reply:
Yep, I will change it in another pr, because I need change the store repositories too